config/install/docker.ci.acceptance.dynamic-store-off.robot.yml

```diff
@@ -114,25 +120,16 @@ sections:
         propel-migration-cleanup:
             command: 'vendor/bin/console propel:migration:delete'
 
-    demodata:
-        queue-setup:
-            command: 'vendor/bin/console queue:setup'
-
         init-database:
             command: 'vendor/bin/console setup:init-db'
 
+    demodata:
         import-eu-region-demodata:
             command: 'vendor/bin/console data:import --config=data/import/local/full_EU.yml'
 
         apply-scheduled-prices:
             command: 'vendor/bin/console price-product-schedule:apply'
 
-        setup-search-create-sources:
-            command: 'vendor/bin/console search:setup:sources'
-
-        propel-copy-schema:
-            command: 'vendor/bin/console propel:schema:copy'
```

Comment:
Don't we longer need it? (Similar issue in other files)

Reply:
Same here https://github.com/spryker-shop/suite/pull/1597/files#r1761053767